app/models/list.rb

```diff
@@ -0,0 +1,10 @@
+class List < ActiveRecord::Base
+  has_many :list_entries
```

Comment:
Maybe we can add a "dependent: :destroy" here to prevent inconsistency on List Entries table. What do you think?